payments-core/src/main/java/com/stripe/android/paymentsheet/model/SupportedSavedPaymentMethod.kt

```diff
@@ -0,0 +1,71 @@
+package com.stripe.android.paymentsheet.model
+
+import android.content.res.Resources
+import androidx.annotation.DrawableRes
+import com.stripe.android.R
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.PaymentMethod
+
+/**
+ * Enum defining all types of saved payment method supported on Payment Sheet.
+ *
+ * These are fetched from the
+ * [PaymentMethods API endpoint](https://stripe.com/docs/api/payment_methods/list) for returning
+ * customers.
+ */
+internal enum class SupportedSavedPaymentMethod(
```

Comment:
It looks like the api supports sending sofort for a payment method type when retrieving a custom: https://stripe.com/docs/api/payment_methods/list.  Should we be requesting those as well?

Reply:
I initially added `sofort` and `bancontact`, but then removed because they become `sepa_debit`, so we can't verify that they work. Maybe they are supported in the API call for historic reasons, but if you save them and query for their types it'll return empty.